lib/admin/components/UserAdmin.js

```diff
@@ -1,59 +1,76 @@
 // @flow
 
 import Icon from '@conveyal/woonerf/components/icon'
-import React, {Component} from 'react'
+import * as React from 'react'
 import {
   Grid,
   Row,
   Col,
   Panel,
   ListGroup,
-  ListGroupItem,
+  ListGroupItem as LGI,
   Button
 } from 'react-bootstrap'
 import {browserHistory} from 'react-router'
 import {LinkContainer} from 'react-router-bootstrap'
 
 import * as adminActions from '../actions/admin'
 import * as organizationActions from '../actions/organizations'
+import Loading from '../../common/components/Loading'
 import ManagerPage from '../../common/components/ManagerPage'
 import {getComponentMessages, isModuleEnabled} from '../../common/util/config'
 import * as feedActions from '../../manager/actions/feeds'
 import * as projectActions from '../../manager/actions/projects'
 import * as managerUserActions from '../../manager/actions/user'
 import OrganizationList from './OrganizationList'
+import ServerSettings from './ServerSettings'
 import UserList from './UserList'
 
 import type {Props as ContainerProps} from '../containers/ActiveUserAdmin'
-import type {Project} from '../../types'
-import type {AdminUsersState, OrganizationsState, ManagerUserState} from '../../types/reducers'
+import type {OtpServer, Project} from '../../types'
+import type {
+  AdminUsersState,
+  OrganizationsState,
+  ManagerUserState
+} from '../../types/reducers'
 
 type Props = ContainerProps & {
   activeComponent: string,
   createOrganization: typeof organizationActions.createOrganization,
   createUser: typeof adminActions.createUser,
   deleteOrganization: typeof organizationActions.deleteOrganization,
+  deleteServer: typeof adminActions.deleteServer,
   deleteUser: typeof adminActions.deleteUser,
   fetchOrganizations: typeof organizationActions.fetchOrganizations,
   fetchProjectFeeds: typeof feedActions.fetchProjectFeeds,
   fetchProjects: typeof projectActions.fetchProjects,
+  fetchServers: typeof adminActions.fetchServers,
   fetchUsers: typeof adminActions.fetchUsers,
   isFetching: boolean,
   organizations: OrganizationsState,
+  otpServers: Array<OtpServer>,
   projects: Array<Project>,
   setUserPage: typeof adminActions.setUserPage,
   setUserQueryString: typeof adminActions.setUserQueryString,
   updateOrganization: typeof organizationActions.updateOrganization,
+  updateServer: typeof adminActions.updateServer,
```

Comment:
Way too many props that aren't needed directly in this component. This needs a refactor so that some of the child components in this component become connected components so that we can avoid passing through so many props.